### ISSUE TITLE
Error: binary architecture "arm64" does not match system architecture "aarch64"

### ISSUE THREAD

**Issue by Transfusion** [confluentinc/cli#2917]
Tried to run Kafka in Docker according to https://developer.confluent.io/quickstart/kafka-local/ for learning purposes with the latest glibc arm64 binary https://github.com/confluentinc/cli/releases/download/v4.9.0/confluent_4.9.0_linux_arm64 and ran into this error.

```
% uname -a
Linux fedora 6.11.4-201.fc40.aarch64 #1 SMP PREEMPT_DYNAMIC Sun Oct 20 15:16:03 UTC 2024 aarch64 GNU/Linux
```

```
% docker version
Client: Docker Engine - Community
 Version:           27.3.1
 API version:       1.47
 Go version:        go1.22.7
 Git commit:        ce12230
 Built:             Fri Sep 20 11:42:06 2024
 OS/Arch:           linux/arm64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          27.3.1
  API version:      1.47 (minimum version 1.24)
  Go version:       go1.22.7
  Git commit:       41ca978
  Built:            Fri Sep 20 11:40:52 2024
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.7.22
  GitCommit:        7f7fdf5fed64eb6a7caf99b3e12efcf9d60e311c
 runc:
  Version:          1.1.14
  GitCommit:        v1.1.14-0-g2c9f560
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0

% docker info
Client: Docker Engine - Community
 Version:    27.3.1
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.17.1
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.29.7
    Path:     /usr/libexec/docker/cli-plugins/docker-compose
...
Kernel Version: 6.11.4-201.fc40.aarch64
 Operating System: Fedora Linux 40 (Forty)
 OSType: linux
 Architecture: aarch64
 CPUs: 4
 Total Memory: 23.41GiB

```

```
% ./confluent/confluent local kafka start                                           
The local commands are intended for a single-node development environment only, NOT for production usage. See more: https://docs.confluent.io/current/cli/index.html

Error: binary architecture "arm64" does not match system architecture "aarch64"
```

**Comment by sgagniere** [comment]
Hi, are you able to run other commands besides kafka local?

**Comment by cqin-confluent** [comment]
Hi, this issue arised likely due to the Confluent CLI binary you are using is built for "arm64", while your system identifies its architecture as "aarch64". Since you are on Fedora, you should use the RPM package specifically built for aarch64: [confluent-cli-4.13.0-1.aarch64.rpm](https://github.com/confluentinc/cli/releases/download/v4.13.0/confluent-cli-4.13.0-1.aarch64.rpm). 
Could you please confirm whether the `confluent local kafka start` command is working after using the aarch64 package?

**Comment by cqin-confluent** [comment]
Hi, we have implemented a fix, and this bug should be gone by the next Confluent CLI release.

**Comment by cqin-confluent** [comment]
Hi, this fix is included in the latest CLI release [v4.13.1](https://github.com/confluentinc/cli/releases/tag/v4.13.1). Closing this issue
